tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -112,7 +112,8 @@ def tap_comment(self):
         element.tap()
         self.marionette.execute_script(
             'arguments[0].scrollIntoView(true);', [element])
-        element = self.marionette.find_element(*self._comment_locator)
+        comment = self.marionette.find_element(*self._comment_locator)
+        Wait(self.marionette, timeout=120).until(expected.element_displayed(comment))
         self.marionette.execute_script(
             'arguments[0].scrollIntoView(true);', [element])
         element.tap()
```

Comment:
actually, when martijn's PR is merged, this would be deleted anyways, so you don't need to fix it.  See https://github.com/mozilla-b2g/gaia/pull/32035